src/components/Input/Input.tsx

```diff
@@ -5,8 +5,8 @@ import { InputProps } from './types';
 
 import { useInput } from './useInput';
 
-import EyeIcon from '../../assets/icons/eye.svg';
-import EyeCrossedIcon from '../../assets/icons/eyeCrossed.svg';
+import EyeIcon from '@assets/icons/eye.svg';
+import EyeCrossedIcon from '@assets/icons/eyeCrossed.svg';
 import styles from './Input.css';
```

Comment:
🧐
Имеет смысл этот отделять
```suggestion
import EyeCrossedIcon from '@assets/icons/eyeCrossed.svg';

import styles from './Input.css';
```

Reply:
ОК. В следующем ПР исправлю во всех файлах.